docs/admin/auth.md

```diff
@@ -138,6 +136,74 @@ To upgrade Coder, run:
 helm upgrade <release-name> coder-v2/coder -n <namespace> -f values.yaml
 ```
 
+### Step 2 (JWT/PKI/Certificate Auth): Configure Coder with the OpenID Connect credentials
+
+<blockquote class="warning">
+  <p>
+  Only <b>Azure AD</b> has been tested with this method. Other OIDC providers may not work, as most providers add additional requirements ontop of the standard that must be implemented. If you are using another provider and run into issues, please leave an issue on our <a href="https://github.com/coder/coder/issues">Github</a>.
+  </p>
+</blockquote>
```

Comment:
I noticed that sometimes we use these HTML blockquotes and sometimes GitHub Style blockquote.

Reply:
The other method is I am not sure if we support. I asked Bruno if he can add it.